crates/cdk-sqlite/src/wallet/mod.rs

```diff
@@ -104,6 +104,7 @@ impl WalletDatabase for WalletSqliteDatabase {
                     nuts,
                     mint_icon_url,
                     motd,
+                    time: _,
```

Comment:
I think we should save the time here as it serves as an indication of when the `MintInfo` was last fetched.

Reply:
addressed in 347b8a19caf955f87174e3b389e3069794875078